.github/workflows/build_docs.yaml

```diff
@@ -0,0 +1,48 @@
+name: Build Documentation
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build Docs
+    runs-on: ubuntu-latest
+    defaults:
+     run:
+       # Adding -l {0} helps ensure conda can be found properly.
+       shell: bash -l {0}
+    env:
+      ENV_NAME: docs
+      PYTHON: 3.7
+    strategy:
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+
+      - name: Setup Miniconda
+        uses: conda-incubator/setup-miniconda@v2.0.0
```

Comment:
we could change this from v2.0.0 to master. :thinking: but then I worry about what if a feature is added/changed and messes with workflows? Really that isn't such a big deal because we can make a quick fix PR. Thoughts?

Reply:
I could go either way, but I'd vote to make it a new PR to change to master if that's ok.